backend/app/ingestion/phase2/IngestStorePolling.scala

```diff
@@ -88,7 +88,7 @@ class IngestStorePolling(
       )
       pollCompleteFuture.onComplete {
         case Success(pollDuration) => schedulePoll(pollDuration)
-        case SFailure(NonFatal(t)) =>
+        case SFailure(t) =>
```

Comment:
The 2.13 compiler is now smart enough to know this wasn't an exhaustive match as it would fail on a fatal error but tbh what chance do we have in that situation (out of memory etc) anyway. So I just fixed the warning by widening the match.